common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -197,27 +192,12 @@ class CapaFields(object):
                                 scope=Scope.user_state)
     done = Boolean(help=_("Whether the student has answered the problem"), scope=Scope.user_state)
     seed = Integer(help=_("Random seed for this student"), scope=Scope.user_state)
-    minutes_allowed = IntegerWithWarningField(
-        display_name=('Minutes Allowed'),
-        help=_(
-            'Number of minutes allowed to finish this assessment. Set 0 for no time-limit. '
-            'If there is a time-limit, the student will only be given one attempt.'
-        ),
-        warning=_(
-            'Setting minutes allowed means that this question can only have one attempt, '
-            'regardless of the value of the maximum attempts field.'
-        ),
-        default=0,
-        scope=Scope.settings,
-    )
-    time_started = Date(help=_("time student started this assessment"), scope=Scope.user_state)
     last_submission_time = Date(help=_("Last submission time"), scope=Scope.user_state)
     submission_wait_seconds = Integer(
         display_name=_("Timer Between Attempts"),
         help=_("Seconds a student must wait between submissions for a problem with multiple attempts."),
         scope=Scope.settings,
-        default=0,
-    )
+        default=0)
```

Comment:
This is just to match upstream.

Reply:
unfortunate formatting incongruity from upstream

Reply:
Yeah, this is one of those times where we got a little ahead of ourselves cleaning up :)